### ISSUE TITLE
Is the abundance of the CAG the mean/sum of the ASVs in the CAG？

### ISSUE THREAD

**Issue by shimw6828** [crtsjtu/AELC#2]
Are the values in "cag abundance.xlsx" the average/sum of the ASVs? How to calculate the cag abundance?

**Comment by crtsjtu** [comment]
> Are the values in "cag abundance.xlsx" the average/sum of the ASVs? How to calculate the cag abundance?

Yes, the CAG abundance is the sum of ASVs belonging to that CAG.

**Comment by shimw6828** [comment]
> 

Thank you! I have an another question. How to get the CAGs? ANN cluster or Canopy cluster? The paper mentions that the cutoff is R > 0.4. A is related to B. B is related to C. C is not related to C. Are the three a CAG in this case?
